gamestore/settings.py

```diff
@@ -106,7 +106,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
```

Comment:
I would recommend to use UTC: https://docs.djangoproject.com/en/3.1/topics/i18n/timezones/